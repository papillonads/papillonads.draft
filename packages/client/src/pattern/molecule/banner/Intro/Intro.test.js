import introDefault from './Intro.render'

describe('<Intro />', () => {
  describe('Snaphot', () => {
    test('must match introDefault', () => {
      expect(global.renderToJSON(introDefault)).toMatchSnapshot()
    })
  })
})
