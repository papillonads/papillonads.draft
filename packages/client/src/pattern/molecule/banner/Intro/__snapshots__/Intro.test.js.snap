// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Intro /> Snaphot must match introDefault 1`] = `
<div
  className="md-order-1 d-flex flex-lg-column flex-auto flex-wrap"
>
  <div
    className="flex-row width-full bg-blue py-3 py-lg-6"
  >
    <div
      className="main-content mx-auto py-4 px-3 px-md-6 px-lg-3 text-white"
    >
      <div
        className=""
      >
        <div />
      </div>
    </div>
  </div>
  <div
    className="flex-row main-content my-6 mx-auto px-3 px-md-6 px-lg-3"
  />
</div>
`;
